extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -70,6 +70,18 @@ export function trimChars(input: string, chars: string): string {
 	return output;
 }
 
+/**
+ * Ensures that folder path terminates with the slash.
+ * Windows-style slash (`\`) is used by default.
```

Comment:
Hm... what does that "standardization" apply to? Are you suggesting I should also change `convertSlashesForSqlProj` to use `/` instead of `\`? The only use case for this method currently is to ensure trailing slash when comparing entries in SQL proj, thus the default. I assume some methods are used in other scenarios, but I don't quite follow the standardization.

Reply:
Ah, I must've had it mixed up; `convertSlashesForSqlProj()` is normalizing in the intended direction.  Makes sense, 'cause we can't break SSDT back compat on Windows, but can do conversions on the fly in this new code.